src/main/target/common_defaults_post.h

```diff
@@ -257,3 +257,18 @@
 #else
 #define MAX_GYRODEV_COUNT 1
 #endif
+
+#ifdef USE_VCP
+#ifndef USB_DETECT_PIN
+#define USB_DETECT_PIN NONE
+#endif
+#endif
+
+#ifndef USB_MSC_BUTTON_PIN
+#define USB_MSC_BUTTON_PIN NONE
+#endif
+#if defined(USE_USB_MSC_BUTTON_IPU) && !defined(MSC_BUTTON_IPU)
+#define MSC_BUTTON_IPU true
+#else
+#define MSC_BUTTON_IPU false
```

Comment:
Should this be defined even for no `USE_` clause?

Reply:
Yeah, right. Should be inside `USE_VCP`.

Reply:
I think this whole conditional is broken. It selects `#else` clause if `USE_USB_MSC_BUTTON_IPU` is not defined...